tests/fuzz/Makefile.am

```
@@ -43,6 +43,7 @@ AM_CPPFLAGS                                               = \
 COMMON_LDADD                                              = \
     $(top_builddir)/src/core/libopenthread-ftd.a            \
     $(top_builddir)/third_party/mbedtls/libmbedcrypto.a     \
+	$(top_builddir)/third_party/tcplp/libtcplp.a            \
```

nit: use spaces instead of tab
```suggestion
    $(top_builddir)/third_party/tcplp/libtcplp.a            \
```

---

Thanks for finding this. I've addressed this now in the latest version.